README.md

```diff
@@ -90,6 +90,52 @@ of the analysis. It should follow the format of JSON-schema
 
 Output types are described further [below](#output-types).
 
+## Input Form Layout Endpoint (optional)
+
+Endpoint `/layout/inputs/<task-name>` may provide simplistic specification
+of a form's layout needed to gather computational task input.
```

Comment:
I'd go for the situation, that `layout` is optional, since we can handle the form basing on schema *only* with default styling. The form is present there, even if no styling is applied on it.

I thought about handling `404` as well as empty response, but it may be clarified.

Reply:
This is problematic:
- If we go for `404 Not Found`, misconfigured proxy call to the worker will also yield no configuration form to the user, without raising any concern (yikes!).
- If we go for `204 No Content`, we will catch the above situation but we need to make `/layout/inputs/<task-name>` endpoint mandatory.

Reply:
We agreed to go for `404`.